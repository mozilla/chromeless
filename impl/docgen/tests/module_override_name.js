/**
 * @module some_other_name
 *
 * This module overrides the logic that assumes that the filename
 * *is* the module name.
 */
