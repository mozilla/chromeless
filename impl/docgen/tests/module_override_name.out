{
  "filename": "tests/module_override_name.js", 
  "module": "some_other_name",
  "desc": "This module overrides the logic that assumes that the filename\n*is* the module name."
}
